packages/lib/server/defaultResponder.ts

```diff
@@ -14,13 +14,14 @@ export function defaultResponder<T>(f: Handle<T>) {
       const result = await f(req, res);
       ok = true;
       if (result && !res.writableEnded) {
-        res.json(result);
+        return res.json(result);
       }
     } catch (err) {
       console.error(err);
       const error = getServerErrorFromUnknown(err);
-      res.statusCode = error.statusCode;
-      res.json({ message: error.message });
+      return res
+        .status(error.statusCode)
+        .json({ message: error.message, url: error.url, method: error.method });
```

Comment:
add some information to the error, we now properly get error message when using http error, spreading redactedClause didn't work